src/main/java/core/basesyntax/WaterAnimals.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class WaterAnimals extends Animal {
```

Comment:
it does make sense to make this class abstract as well, cause we don't want to create abstract WaterAnimal